src/common/actions/snaps.js

```diff
@@ -14,85 +11,37 @@ export const REMOVE_SNAP_ERROR = 'REMOVE_SNAP_ERROR';
 
 export function fetchUserSnaps(owner) {
   return async (dispatch) => {
-    const url = `${BASE_URL}/api/launchpad/snaps/list`;
     const query = `owner=${encodeURIComponent(owner)}`;
 
     dispatch({
-      type: FETCH_SNAPS
-    });
-
-    try {
-      const response = await fetch(`${url}?${query}`, {
-        headers: { 'Content-Type': 'application/json' },
-        credentials: 'same-origin'
-      });
-      await checkStatus(response);
-      const result = await response.json();
-      if (result.status !== 'success') {
-        throw getError(response, result);
+      type: FETCH_SNAPS,
+      [CALL_API]: {
+        types: [ FETCH_SNAPS_SUCCESS, FETCH_SNAPS_ERROR ],
```

Comment:
@earnubs The request type is defined on the action instead of in [CALL_API]. If the action doesn't have a type, it breaks some Redux tools like redux-mock-store. Not a lot has changed here, only how the "request" action type is defined.